### PR TITLE
Adding support to remote/network devices on fpicker

### PR DESCRIPTION
**Pull request by diogopim**
In order to fuzz on a remote device (running 'frida-server') I added support using the Frida core options.

Added details to README.

I just noticed some additional 'diff', because my editor is removing trailing white spaces from edited files.

### CODE REVIEW

**Comment by diogopim** [comment]
I also fixed build issues for Linux.

File: fpicker.c [hunk 1]
```diff
@@ -140,6 +140,15 @@ bool check_config(fuzzer_config_t *config) {
             plog("[!] Fuzzer mode AFL not possible with USB device.\n");
             return false;
         }
+    } else if (config->device == DEVICE_REMOTE) {
```
Comment (diogopim) [inline fpicker.c:143]: I used the same constraint I observed above for USB devices. Not sure this applies.

File: harness/stalker-instrumentation.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ if (Process.arch == "x64") {
   pc = "rip";
 } else if (Process.arch == "arm64") {
   pc = "pc";
+} else if (Process.arch == "arm") {
```
Comment (diogopim) [inline harness/stalker-instrumentation.js:7]: This fixes an issue with missing 32 bit ARM architecture support.

**Comment by ttdennis** [comment]
Hey, thanks for your contribution! That looks really nice. 

> I just noticed some additional 'diff', because my editor is removing trailing white spaces from edited files.

Ouch, thanks for catching that! Not sure how these got added...

> I also fixed build issues for Linux.

Hmm, the reason I had the differentiation between the `-linux`, `-ios`, and `-macos` headers and libs is that I often build the fuzzer out of the same directory for the different platforms. It's not really an elegant solution but it at least keeps me from having to copy and paste the different architecture libs into the directory, so I reverted that back.

**Comment by diogopim** [comment]
Understood, I'll keep the Linux changes to myself.
BTW, thanks for the tool. It's working great.

**Pull request merged**